core-bundle/src/Security/Voter/BackendAccessVoter.php

```diff
@@ -94,28 +116,98 @@ private function hasAccess($subject, string $field, BackendUser $user): bool
     private function isAllowed($subject, int $flag, BackendUser $user): bool
     {
         if ($subject instanceof PageModel) {
-            $subject->loadDetails();
             $subject = $subject->row();
         }
 
         if (!\is_array($subject)) {
-            return false;
+            $page = $this->framework->getAdapter(PageModel::class)->findByPk($subject);
+
+            if (!$page instanceof PageModel) {
+                return false;
+            }
+
+            $subject = $page->row();
```

Comment:
This looks strange to me. The same code is already on line 119. Do we have to remove lines 119–120?

Reply:
The first case handles a PageModel being passed. The second one supports loading the pagemodel if an ID is given.